### PR TITLE
[16.0][MIG] account_fiscal_position_type (from 12.0 to 16.0)

### PR DESCRIPTION
**Pull request by legalsylvain**
### Description

This module extends the functionality of account to add a use type on fiscal position. (``sale``, ``purchase`` or ``all``) to restrict the usage of fiscal position for out or in invoices.

* If a fiscal position is configured for sale, it will not be possible to use it on a customer invoice

* If a fiscal position is configured for purchase, it will not be possible to use it on a supplier invoice

### Configure

To configure this module, you need to:

* Go to Invoicing / Configuration / Taxes / Fiscal Positions

* On a fiscal position, set the correct settings.

fiscal_position_form

**Note**

It is possible to set the value on the fiscal position templates. In that case, installing a new chart of account will configure correctly the fiscal positions.

### CODE REVIEW

**Comment by rafaelbn** [comment]
Hello @legalsylvain ,

Tested and it works. I think this is the opposite

>     If a fiscal position is configured for sale, it will not be possible to use it on a customer invoice
If a fiscal position is configured for sale, it will not be possible to use it on a vendor bill
>     If a fiscal position is configured for purchase, it will not be possible to use it on a supplier invoice
If a fiscal position is configured for purchase, it will not be possible to use it on a customer invoice

imagen

**Review by rafaelbn — COMMENTED** [review]
Tested, just IMHO fix readme 😄 👍

**Comment by rafaelbn** [comment]
In my dreams you can set for `res.partner` diferent fiscal position for sales and purchases 😄

**Review by rafaelbn — APPROVED** [review]
README is correct! 👍 

* If a fiscal position is configured for sale, it will not be possible to use
  it in a purchase invoice.

* If a fiscal position is configured for sale, it will not be possible to
  set it to a partner flagged only as 'supplier'

File: account_fiscal_position_type/README.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module extends the functionality of account to add a use type on
+fiscal position. (``sale``, ``purchase`` or ``all``) to restrict the
+usage of fiscal position for out or in invoices.
+
+* If a fiscal position is configured for sale, it will not be possible to use
+  it in a purchase invoice.
```
Comment (rafaelbn) [inline account_fiscal_position_type/README.rst:33]: Odoo terminology is "Vendor bill". Just details.

**Comment by legalsylvain** [comment]
Hi @rafaelbn. Thanks for your review !
- I changed the text for that one : 

```
This module extends the functionality of account to add a use type on
fiscal position. (``sale``, ``purchase`` or ``all``) to restrict the
usage of fiscal position for out or in invoices.

* If a fiscal position is configured for sale, it will not be possible to use it
  on Vendor Bills, Refunds and Purchase Receipts.

* If a fiscal position is configured for purchase, it will not be possible to use it
  on Customer Invoices, Credit Notes and Sales Receipts.

```

- also I added comment in the commit text.

thanks !

**Review by ghost — APPROVED** [review]
Tested! LGTM!

**Comment by SoniaViciana** [comment]
Hi @dreispt,

This module is merged, however the translations are locked:

[account-fiscal-rule-16.0/account-fiscal-rule-16.0-account_fiscal_position_type](https://translation.odoo-community.org/projects/account-fiscal-rule-16-0/account-fiscal-rule-16-0-account_fiscal_position_type/)

Can I know why?

Thanks in advance,

**Comment by kikopeiro** [comment]
> Hi @dreispt,
> 
> This module is merged, however the translations are locked:
> 
> [account-fiscal-rule-16.0/account-fiscal-rule-16.0-account_fiscal_position_type](https://translation.odoo-community.org/projects/account-fiscal-rule-16-0/account-fiscal-rule-16-0-account_fiscal_position_type/)
> 
> Can I know why?
> 
> Thanks in advance,

CC @rafaelbn

**Pull request merged**